docs/_docs/localization/index.md

```diff
@@ -212,25 +222,85 @@ strings xml file and will not be fetched by DesignCompose.
 The plugin also provides a **Clear String Resource** tool to clear the string resource names, char
 limits and descriptions from the plugin data.
 
-### 3.2 It supports mixed styles
+#### 2.3.2 It supports mixed styles
 
 For a text node with a single style, the plugin generates a string resource. And for a text node with
 mixed styles, the plugin generates a string array resource with each item mapping to the text segments
 with different styles.
 
-## 4. What is next
+## 3. Image resource tool
+
+Images play an important role in the figma design. Figma supports PNG, JPG and GIF files up to 4k
+in width and height as background fills or strokes. In some cases, images need localization too,
+for example, the stop sign looks different in different countries.
+
+As a supplemental tool to the string resource tool, it works in a similar way that the plugin goes
+through all the images, allows user to assign a proper resource name to the image which in the
+Android resource framework is the image file name and it writes the resource name to the figma
+plugin data which the DesignCompose uses later to load the image resource.
```

Comment:
Suggestion: "As a supplemental tool to the string resource tool, the image resource tool works in a similar way. It goes through all the images and allows a user to assign a proper resource name to the image to match the image file name in the Android resource framework. It then writes the resource name to the Figma plugin data, which DesignCompose uses later to load the image resource."